racetime/forms.py

```diff
@@ -85,15 +86,24 @@ class CategoryForm(forms.ModelForm):
     ALLOWED_TAGS = {
         'a': ('href', 'title'),
         'b': (),
+        'br': (),
+        'blockquote': (),
+        'code': (),
         'em': (),
         'i': (),
+        'h1': (),
+        'h2': (),
         'h3': (),
         'h4': (),
         'h5': (),
         'h6': (),
+        'hr': (),
+        'li': (),
+        'ol': (),
         'p': (),
         's': (),
         'u': (),
+        'ul': (),
```

Comment:
There ought to be some styling for these new elements if we're going to allow them in the info box. Particularly the lists, which right now don't render properly at all because by default `ol`/`ul` get clobbered with `list-style: none` (perhaps not the best idea to begin with…). `h1` and `h2` should also be tidied up so they don't upstage the category header.

Reply:
So I've done a bit of a cheat here. If this recent change doesn't appeal to you I'd suggest you take the styling. I've just reverted all of the child items to standard as because this is "markdown" I'm leaning to say its up to them how to really deal with it. For now I've just disabled the weight of `h1` and `h2` in addition to that revert to prevent the upstage.